File: cookiecutter-spring-mvn-template/{{cookiecutter.project_slug}}/src/main/java/{{cookiecutter.package_path}}/application/service/DefaultGreetingService.java
package {{cookiecutter.package_name}}.application.service;

import {{cookiecutter.package_name}}.domain.model.Greeting;
import {{cookiecutter.package_name}}.domain.service.GreetingService;
import org.springframework.stereotype.Service;

@Service
public class DefaultGreetingService implements GreetingService {

    @Override
    public Greeting greet(String name) {
        return new Greeting(name);
    }
}

File: cookiecutter-dotnet-template/{{cookiecutter.project_slug}}/Application/Services/DefaultGreetingService.cs
using {{cookiecutter.namespace}}.Domain.Models;
using {{cookiecutter.namespace}}.Domain.Services;

namespace {{cookiecutter.namespace}}.Application.Services
{
    public class DefaultGreetingService : IGreetingService
    {
        public Greeting Greet(string? name)
        {
            return new Greeting(name);
        }
    }
}
